dartagnan/src/main/java/com/dat3m/dartagnan/parsers/program/visitors/boogie/StdProcedures.java

```diff
@@ -45,19 +42,6 @@
 	
 	public static void handleStdFunction(VisitorBoogie visitor, Call_cmdContext ctx) {
 		String name = ctx.call_params().Define() == null ? ctx.call_params().Ident(0).getText() : ctx.call_params().Ident(1).getText();
-		if(name.startsWith("WRITE_ONCE")) {
-			IExpr address = (IExpr)ctx.call_params().exprs().expr(0).accept(visitor);
-			ExprInterface value = (ExprInterface)ctx.call_params().exprs().expr(1).accept(visitor);
-			visitor.programBuilder.addChild(visitor.threadCount, EventFactory.newStore(address, value, "NA"));
-			return;
-		}
-//		if(name.startsWith("READ_ONCE")) {
-//			System.out.println(ctx.getText());
-//			IExpr address = (IExpr)ctx.call_params().exprs().expr(0).accept(visitor);
-//			ExprInterface value = (ExprInterface)ctx.call_params().exprs().expr(1).accept(visitor);
-//			visitor.programBuilder.addChild(visitor.threadCount, new Store(address, value, "NA"));
-//			return;
-//		}
```

Comment:
What do have these changes to do with the PR?

Reply:
Nothing, that code was related to the thesis of one bachelor student but which we will not integrate and thus we can get rid of this. If just made it here because only here I noticed.